### PR TITLE
add entry for associated chat room

### PR DESCRIPTION
**Pull request by affeldt-aist**
For projects from outside community.

As suggested in https://github.com/math-comp/analysis/pull/297 @Zimmi48

### CODE REVIEW

File: README.md.mustache [hunk 1]
```diff
@@ -6,6 +6,9 @@
 {{# action }}
 [![CI][action-shield]][action-link]
 {{/ action }}
+{{# chat }}
+[![CHAT][chat-shield]][chat-link]
```
Comment (Zimmi48) [inline README.md.mustache:10]: The link title doesn't have to be all caps.

File: ref.yml [hunk 2]
```diff
@@ -107,6 +107,18 @@ fields:
...
+        Url to the chat room associated with this project.
+      required: false
+      used:
+        - README.md
+      item_fields:
+        - shield:
+            description: >
+              Badge associated with the website of the chat room.
+            default: ""
+            examples: zulip, matrix, discourse
```
Comment (Zimmi48) [inline ref.yml:121]: The `url` field item is missing. Besides, I would change the description to something more precise:
- for the `chat` variable: adds a badge with a link to a chat room
- for the `shield` variable: name of the chat to display in the shield icon.

Comment (affeldt-aist) [reply ref.yml]: > The url field item is missing.

Oops. I wonder why it worked ^_^. Sorry about that, the semantics is not yet clear to me.

Comment (Zimmi48) [reply ref.yml]: Well, `ref.yml` is not actually checked against anything, so...

**Review by Zimmi48 — APPROVED** [review]

**Comment by Zimmi48** [comment]
This looks ready to me. Did you test that it works well in your use case?

**Comment by affeldt-aist** [comment]
> This looks ready to me. Did you test that it works well in your use case?

Yes, I did. The output README.md looks ok to me and its html version is rendered fine by pandoc/firefox.

**Pull request merged**